example-config.json

```diff
@@ -9,7 +9,8 @@
             "keyspace":"clio",
             "replication_factor":1,
             "table_prefix":"",
-            "max_requests_outstanding":25000,
```

Comment:
I don't know how to avoid renaming this _particularly important_ config option. 

Could we leave `max_requests_outstanding` as an option, and add `max_write_requests_outstanding` as an alias for this config option?

Reply:
I feel like its ok to rename config options, because operators choose when they update, and the changes are in the release notes. I think two config options for the same thing is sloppy, and then its unclear which takes precedence. But I'm not totally sure.